jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/MavenWebAppContext.java

```diff
@@ -219,16 +219,18 @@ public boolean getBaseAppFirst()
      */
     public void setResourceBases(String[] resourceBases)
     {
-        List<String> resources = new ArrayList<String>();
-        for (String rl : resourceBases)
+        try
         {
-            String[] rs = StringUtil.csvSplit(rl);
-            for (String r : rs)
-            {
-                resources.add(r);
-            }
+            List<Resource> resources = Resource.fromList(List.of(resourceBases), false);
```

Comment:
This looks broken, as trying to create a `Resource` out of a URI that needs to be mounted throws.

Reply:
This `resourceBases` is user space provided.
It can be simple paths (absolute or relative), or URI references (or a combination of both)

Reply:
I believe the latest cleanup has addressed this concern.